boilerplate/storybook/storybook.tsx

```diff
@@ -12,6 +12,11 @@ const StorybookUI = getStorybookUI({ port: 9001, host: "localhost", onDeviceUI:
 export class StorybookUIRoot extends React.Component {
   componentDidMount() {
     SplashScreen.hide()
+    if (typeof __TEST__ === "undefined" || !__TEST__) {
```

Comment:
`undefined` is falsy, so you could use `!__TEST__`.

Reply:
You would think that, but it was causing explosions, hence this solution.

Reply:
Interesting. So leaving this as `if (typeof __TEST__ === "undefined" || !__TEST__)`

Reply:
To be fair, it was exploding prior to declaring __TEST__ in setup.ts, but I'm pretty sure I tested removing this after that and still faced issues.